packages/frontend/src/features/post/components/Post/Post.tsx

```diff
@@ -193,6 +187,8 @@ export default function Post({
             downCount: downCount,
             isMine: isMine,
             finalAction: finalAction,
+            votedNonce: votedNonce,
+            votedEpoch: votedEpoch,
```

Comment:
This useEffect is missing dependencies: `votedNonce` and `votedEpoch`.

Reply:
fixed